pybv/tests/test_bv_writer.py

```diff
@@ -21,16 +21,16 @@
 
 from pybv.io import write_brainvision, _write_bveeg_file, _write_vhdr_file
 
-# Create data we'll use for testing
+# create testing data
 fname = 'pybv'
 np.random.seed(1337)
 n_chans = 10
-ch_names = ['ch_{}'.format(ii) for ii in range(n_chans)]
-sfreq = 1000.
+ch_names = ['ch_{}'.format(i) for i in range(n_chans)]
```

Comment:
I was once told that single letter variables are not so great, because they are hard to search for in code ... that's why I started to use `ii` instead of `i` :man_shrugging: 

maybe that rule I said is not so important for list comprehensions and loop indices. It's fine with me to change it here.

Reply:
`grep`ing for variable names is asking for trouble and AFAIK even VSCode without any plugins properly highlights single letter variables. OTOH, if you're looking for expressive variable names and `i` isn't good enough, `index` is better than `ii`.

Reply:
I agree with @tstenner, in some cases it's fine to use one-letter names, and in other cases more expressive names are needed (and `ii` is not more expressive than `i`).

Reply:
thanks for the explanations :+1: